template/src/pages/AppErrorPage.tsx

```diff
@@ -0,0 +1,11 @@
+import ErrorState from '@rio-cloud/rio-uikit/ErrorState';
+
+const AppErrorPage = () => (
+    <ErrorState
+        outerClassName="margin-top-20"
+        headline="Something went very wrong"
+        message="We're sorry, the application encountered an unexpected error. Please try again later."
```

Comment:
See Phrase chared project "common messages" comment for RoutingErrorPage

Reply:
That won't work because we're outside of the `IntlProvider` here.